src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -345,8 +345,8 @@ public function symlink($originDir, $targetDir, $copyOnWindows = false)
     /**
      * Given an existing path, convert it to a path relative to a given starting path.
      *
-     * @param string $endPath   Absolute path of target
-     * @param string $startPath Absolute path where traversal begins
+     * @param string $endPath   Path of target
+     * @param string $startPath Path where traversal begins
```

Comment:
Great. Are there any other changes I should make to the pull request? If not, it would be great if you could update your review status from “requested changes” to “approved” :)